app.js

```diff
@@ -17,13 +17,18 @@ import paymentRouter from "./routes/payment";
 import conversationRouter from "./routes/conversation";
 
 var app = express();
+const path = require("path")
 
 app.use(logger("dev"));
 app.use(json());
 app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(cors());
-app.use(express.static(join(__dirname, "public")));
+app.use(express.static(path.join(__dirname, "client", "build")));
+
+app.get("*", (req, res) => {
+  res.sendFile(path.join(__dirname, "client", "build"));
+});
```

Comment:
I believe you need to remove this part for the get requests to work

Reply:
Removing the last bit of this code got the get requests to work as it should. I also noticed removing this whole code works as well, so I'll just delete this code